src/hyperion/experiment_plans/flyscan_xray_centre_plan.py

```diff
@@ -163,6 +165,48 @@ def tidy_up_plans(fgs_composite: FlyScanXRayCentreComposite):
     )  # make sure we don't consume any other results
 
 
+def kickoff_and_complete_gridscan(
+    gridscan: PandaOrZebraGridscan,
+    eiger: EigerDetector,
+    synchrotron: Synchrotron,
+    zocalo_environment: str,
+):
+    @TRACER.start_as_current_span(DO_FGS)
+    @bpp.set_run_key_decorator(DO_FGS)
+    @bpp.run_decorator(
+        md={
+            "subplan_name": DO_FGS,
+            "zocalo_environment": zocalo_environment,
+        }
+    )
+    @bpp.contingency_decorator(
+        except_plan=lambda e: (yield from bps.stop(eiger)),
+        else_plan=lambda: (yield from bps.unstage(eiger)),
+    )
+    def do_fgs():
+        # Check topup gate
+        expected_images = yield from bps.rd(gridscan.expected_images)
+        exposure_sec_per_image = yield from bps.rd(eiger.cam.acquire_time)
```

Comment:
It would be nice to have the `acquire_time` signal in dodal be renamed `acquire_time_s`

Reply:
Agree but I think cleaner to do in it's own PR. Have made https://github.com/DiamondLightSource/hyperion/issues/1188